src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/PolymorphicNuMT.java

```diff
@@ -0,0 +1,90 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator;
+
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.utils.help.HelpConstants;
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.GenotypeBuilder;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.vcf.VCFFormatHeaderLine;
+import org.apache.commons.math3.distribution.PoissonDistribution;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.utils.GATKProtectedVariantContextUtils;
+import org.broadinstitute.hellbender.utils.MathUtils;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.genotyper.ReadLikelihoods;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
+import org.broadinstitute.hellbender.utils.logging.OneShotLogger;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFHeaderLines;
+import org.spark_project.guava.collect.Range;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Potential polymorphic NuMT annotation compares the number of alts to the autosomal coverage
+ *
+ * <p>Polymorphic NuMTs (NuMT sequence that is not in the reference) will result in autosomal reads that map to the
+ * mitochondria. This annotation notes when the number of alt reads falls within 80% of the autosomal coverage
+ * distribution, assuming that autosomal coverage is a Poisson distribution given a central statistic of the depth
+ * (median is recommended). This will also include true low allele fraction mitochondrial variants and should be used
+ * as an annotation, rather than a filter.</p>
+ *
+ * <h3>Caveat</h3>
+ * <p>This annotation can only be calculated in Mutect2 if median-autosomal-coverage argument is provided.</p>
+ *
+ */
+@DocumentedFeature(groupName= HelpConstants.DOC_CAT_ANNOTATORS, groupSummary=HelpConstants.DOC_CAT_ANNOTATORS_SUMMARY, summary="Number of alts indicates it could be an autosomal false positive.")
+public class PolymorphicNuMT extends GenotypeAnnotation implements Annotation {
+    protected final OneShotLogger warning = new OneShotLogger(this.getClass());
+    public static final String KEY = GATKVCFConstants.POTENTIAL_POLYMORPHIC_NUMT_KEY;
+    private static final double LOWER_BOUND_PROB = .1;
+    private long minAutosomalHet;
```

Comment:
I think these can all be `final`.

Reply:
And it looks like some of them don't need to be class members at all because they are local to the constructor.